HousingSearchApi/V1/Boundary/Requests/GetAllAssetListRequest.cs

```diff
@@ -31,9 +31,6 @@ public class GetAllAssetListRequest : GetAssetListRequest
         [FromQuery(Name = "contractIsApproved")]
         public string ContractIsApproved { get; set; }
 
-        [FromQuery(Name = "isTemporaryAccomodation")]
```

Comment:
What's the reason for moving this to `HousingSearchRequest`?

Reply:
Mainly because it was thowing testing errors in core if I tried to use it from here. But could be a better way to implement I'm not thinking about?

Reply:
`HousingSearchRequest` should be kept generic without any connection entities it might be used against, so let's have a look how we can resolve this without including this asset/ta specific proiperty to the base search class.

Reply:
As discussed we won't move this propety, but handle it a bit differently in the implementation.